doc/faq.rst

```diff
@@ -244,3 +243,24 @@ program: Insert the following instructions in your main script::
 
 You can find more default on the new start methods in the `multiprocessing
 documentation <https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods>`_.
+
+How do I set a random_state for an entire execution?
+----------------------------------------------------
+
+For testing and replicability, it is often important to have the entire execution
+controlled by a single seed for the pseudo-random number generator used in algorithms
+that have a randomized component. Scikit-learn does not use its own global random state;
+whenever a RandomState instance or an integer random seed is not provided as an argument,
+it relies on the numpy global random state, which can be set using ''numpy.random.seed'' function.
+For example, to set an execution's numpy global random state to 42, one could execute
+the following in his or her script::
+
+    import numpy as np
+    np.random.seed(42)
+
+However, a global random state will not ensure replicability when parallel execution is used
```

Comment:
That not the correct explanation of why a global random state is fragile. The correct explanation is that it is fragile to other code modifying it during the execution.

Discussing parallel computing is not relevant here, as this modification may be in parallel or sequential.